data/dataset_classes/imagenet_2015.txt

```diff
@@ -1,1000 +0,0 @@
-n02119789 kit fox
```

Comment:
I think it can be stayed for opportunity to run somebody converted models outside of omz

Reply:
Reverted